### ISSUE TITLE
Feature Request Charts and visualization

### ISSUE THREAD

**Issue by MrLoverLoverMMMM** [powerpaul17/nc_money#44]
It would be cool if ther would be an overview which can show with charts and other ways of visualization an overview over the data in the accounting system.

**Comment by yehudaclinton** [comment]
maybe it should be integrated with the [analytics app](https://apps.nextcloud.com/apps/analytics)

**Comment by skillman623** [comment]
> maybe it should be integrated with the [analytics app](https://apps.nextcloud.com/apps/analytics)

Instructions on this could be implemented are [here](https://github.com/Rello/analytics/wiki/Register-own-datasource).

**Comment by SafetyIng** [comment]
I have another idea/feature request here:
On the first page of a book, the income and expenditure could also be compared as a total + the balance accordingly.

**Comment by powerpaul17** [comment]
> I have another idea/feature request here:
> On the first page of a book, the income and expenditure could also be compared as a total + the balance accordingly.

Yes, I wanted to add something like that too, but I'm not sure about the exact representation. Do you have an idea or can you maybe post a sketch or something?
